tests/parsers/test.fluent.js

```diff
@@ -20,33 +20,6 @@ choose-download-folder-title =
     parser.parse();
 
     expect(parser.isValid).toEqual(true);
-    expect(parser.parsedData).toEqual({
-      'choose-download-folder-title': {
-        val: [
-          {
-            def: 0,
-            exp: null,
-            type: 'sel',
-            vars: [
-              {
-                key: {
-                  name: 'nominative',
-                  type: 'sym',
-                },
-                val: 'Foo',
-              },
-              {
-                key: {
-                  name: 'accusative',
-                  type: 'sym',
-                },
-                val: 'Foo2',
-              },
-            ],
-          },
-        ],
-      },
-    });
```

Comment:
I primarily removed these since I essentially only copy the upstream tree into `parsedData` and that tree is huge and would bloat up our tests. There is one test that verifies things at a very basic level so I thought that has to suffice.